sunpy/instr/aia.py

```diff
@@ -49,7 +50,14 @@ def aiaprep(aiamap):
         scale = 0.6*u.arcsec # pragma: no cover # can't test this because it needs a full res image
     scale_factor = aiamap.scale.x / scale
 
-    newmap = aiamap.rotate(recenter=True, scale=scale_factor.value, missing=aiamap.min())
+    tempmap = aiamap.rotate(recenter=True, scale=scale_factor.value, missing=aiamap.min())
+
+    # extract center from padded aiamap.rotate output
+    center = np.floor(tempmap.meta['crpix1'])
```

Comment:
just an inline comment above the line will suffice, it's a suble implementation detail that's all.